peps/pep-0667.rst

```diff
@@ -452,12 +453,18 @@ of both the frame and mapping proxy until the next cycle collection.
 
 ``PyEval_GetLocals()`` already creates a cycle, as it returns a borrowed reference.
 
+Implementation
+==============
+
+The implementation is in development as a draft pull request on GitHub ([2]_).
 
 References
 ==========
 
 .. [1] https://bugs.python.org/issue30744
 
+.. [2] https://github.com/python/cpython/pull/115153
+
```

Comment:
Please could you also turn [1] into an inline link as well?

And link it to `https://github.com/python/cpython/issues/74929` instead of the old bugs.python.org.

```suggestion
```